README.md

```diff
@@ -48,9 +48,14 @@ $ make build
 $ ls $GOPATH/bin/terraform-provider-cloudstack
 ```
 Once the build is ready, you have to copy the binary into Terraform locally (version appended).
-On Linux this path is at ~/.terraform.d/plugins, and on Windows at %APPDATA%\terraform.d\plugins.
+
+On Linux and Mac this path is at ~/.terraform.d/plugins,
+On Windows at %APPDATA%\terraform.d\plugins,
+
 ```sh
-$ ls ~/.terraform.d/plugins/registry.terraform.io/cloudstack/cloudstack/0.4.0/linux_amd64/terraform-provider-cloudstack_v0.4.0
+$  cd ~
+$  mkdir -p .terraform.d/plugins/localdomain/provider/cloudstack/0.4.0/linux_amd64
```

Comment:
```suggestion
$  mkdir -p ~/.terraform.d/plugins/localdomain/provider/cloudstack/0.4.0/linux_amd64
```
Would be better to do it this way rather than chaning the current dir the user might be in